test/unit/model/qti/ItemTest.php

```diff
@@ -21,89 +21,80 @@
 
 namespace oat\taoQtiItem\test\unit\mode\qti;
 
-use oat\generis\test\TestCase;
+use common_ext_Extension;
+use common_ext_ExtensionsManager;
+use oat\generis\test\ServiceManagerMockTrait;
 use oat\oatbox\service\ServiceManager;
 use oat\tao\model\service\ApplicationService;
 use oat\taoQtiItem\model\qti\Item;
+use PHPUnit\Framework\TestCase;
 
 class ItemTest extends TestCase
 {
+    use ServiceManagerMockTrait;
+
     private const PRODUCT_NAME = 'TAO';
 
     protected function setUp(): void
     {
         parent::setUp();
 
-        if (!defined('PRODUCT_NAME')) {
-            define('PRODUCT_NAME', self::PRODUCT_NAME);
-        }
-        if (!defined('ROOT_URL')) {
-            define('ROOT_URL', __DIR__ . '/../../../../../');
-        }
-        if (!defined('CONFIG_PATH')) {
-            define('CONFIG_PATH', ROOT_URL . 'config/');
-        }
-        if (!defined('EXTENSION_PATH')) {
-            define('EXTENSION_PATH', ROOT_URL);
-        }
-
-        ServiceManager::getServiceManager()->overload(
-            ApplicationService::SERVICE_ID,
-            $this->createMock(ApplicationService::class)
-        );
-    }
+        define('PRODUCT_NAME', self::PRODUCT_NAME);
```

Comment:
please don't create constant in this way

Reply:
I tried to found how to declare it or import it but can see it in other unit tests. Old tests use just define as I did. Can you give me some exaple?

Reply:
I guess we still define all this constants except `PRODUCT_NAME` in generis bootstrap or where ever so we can move it there as well

Reply:
I talk with @mike-ionut-mihai-sandu-tao and I just need to wrap it with IF

Reply:
> We could move it to a generis bootstrap file.

We decided together not to do that and I will stick to it.